internal/gatewayapi/testdata/securitypolicy-with-oidc-custom-cookies.out.yaml

```diff
@@ -188,6 +188,7 @@ xdsIR:
             name: securitypolicy/envoy-gateway/policy-for-gateway
             provider:
               authorizationEndpoint: https://accounts.google.com/o/oauth2/v2/auth
+              destination: null
```

Comment:
is this a bug ?

Reply:
destination should be set as `omitempty` as it's optional for OIDC provider.